### ISSUE TITLE
RSWG should appear on list of non-WG mailing lists 

### ISSUE THREAD

**Issue by alexisannerossi** [ietf-tools/datatracker#5777]
### Description

This page https://datatracker.ietf.org/list/nonwg should include a link to the RSWG mailing list at https://mailman.rfc-editor.org/mailman/listinfo/rswg

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by rjsparks** [comment]
That view has all manner of problems. Some are related to the postgresql related change to a case-sensitive database query engine. Others are related to how the view is deciding whether a group should be listed on that page (it currently uses the GroupFeature.acts_like_wg attribute).

To that point - we list stream groups in streams. The research groups and the iab programs all have their own pages. The editorial stream should probably follow a parallel construction.

(Yes, right now edm and one RG (cfrg) are listed on that page. cfrg is listed _twice_ because of the above bugs. That will be corrected. Note that other RG are not listed on the page already.)

In short, it's not clear to me that RSWG _should_ appear here. 

The name of the page might need to change to reflect that it means lists that aren't for a group in a stream.

**Comment by alexisannerossi** [comment]
RSAB is on this list already, it would seem RSWG should be as well. Or they should both be elsewhere, I suppose.

**Comment by rjsparks** [comment]
They are already listed in the menus here:
![image](https://github.com/ietf-tools/datatracker/assets/10996692/1c15e2e5-7a43-480e-8b7e-55a2d5dad4a0)

I'll research whether turning the nonwg page into a non-group page makes sense. (Or if another container description is more accurate). nonwg was inherited from long ago and probably has been a misnomer for years.

**Comment by JayDaley** [comment]
I recommend changing the text "RFC Editor" on the main menu to "Editorial Stream".  As this is an 'internal' tool, it is appropriate to use stream terminology

**Comment by JayDaley** [comment]
@rjsparks I believe this issue can be closed as these groups are now shown under "RFC Editor"
